tests/_jax_rng.py

```diff
@@ -0,0 +1,274 @@
+        self,
+        low: int | ArrayLike,
+        high: int | ArrayLike | None = None,
+        size: Size = None,
+        dtype: DTypeLike = int,
+        endpoint: bool = False,
+    ) -> Array:
+        """
+        Return random integers from the "discrete uniform" distribution
+        of the specified dtype.  If *high* is None (the default), then
+        results are from 0 to *low*.
+        """
+        if high is None:
+            low, high = 0, low
+        if endpoint:
+            high = high + 1
+        return randint(self.__key, _s(size), low, high, dtype)
+
+    def random(self, size: Size = None, dtype: DTypeLike = float) -> Array:
+        """Return random floats in the half-open interval [0.0, 1.0)."""
+        return uniform(self.__key, _s(size), dtype)
+
+    def choice(
+        self,
+        a: Array,
+        size: Size = None,
+        replace: bool = True,
+        p: Array | None = None,
+        axis: int = 0,
+    ) -> Array:
+        """Generate a random sample from a given array."""
+        return choice(self.__key, a, _s(size), replace, p, axis)
+
+    def bytes(self, length: int) -> bytes:
+        """Return random bytes."""
+        shape = (length // uint8.dtype.itemsize,)
+        return bits(self.__key, shape, uint8).tobytes()
+
+    def permutation(self, x: int | Array, axis: int = 0) -> Array:
+        """Randomly permute a sequence, or return a permuted range."""
+        return permutation(self.__key, x, axis, False)
+
+    def beta(self, a: RealArray, b: RealArray, size: Size = None) -> Array:
+        """Draw samples from a Beta distribution."""
+        return beta(self.__key, a, b, _s(size))
+
+    def binomial(self, n: RealArray, p: RealArray, size: Size = None) -> Array:
+        """Draw samples from a binomial distribution."""
+        return binomial(self.__key, n, p, _s(size))
+
+    def chisquare(self, df: RealArray, size: Size = None) -> Array:
+        """Draw samples from a chi-square distribution."""
+        return chisquare(self.__key, df, _s(size))
+
+    def dirichlet(self, alpha: RealArray, size: Size = None) -> Array:
+        """Draw samples from the Dirichlet distribution."""
+        return dirichlet(self.__key, alpha, _s(size))
+
+    def exponential(self, scale: RealArray = 1.0, size: Size = None) -> Array:
+        """Draw samples from an exponential distribution."""
+        return array(scale) * exponential(self.__key, _s(size, scale))
+
+    def f(self, dfnum: RealArray, dfden: RealArray, size: Size = None) -> Array:
+        """Draw samples from an F distribution."""
+        return f(self.__key, dfnum, dfden, _s(size))
+
+    def gamma(
+        self,
+        a: RealArray,
+        scale: RealArray = 1.0,
+        size: Size = None,
+    ) -> Array:
+        """Draw samples from a Gamma distribution."""
+        return array(scale) * gamma(self.__key, a, _s(size, a, scale))
+
+    # geometric(p[, size])
+    # Draw samples from the geometric distribution.
```

Comment:
Can be populated as and when we start using these distributions.

Reply:
This might be a good argument against making it a separate package?